index.html

```diff
@@ -0,0 +1,16 @@
+<html>  
+  <head>
+    <script src="script.js"></script>
+  </head>
+    <body style="background-color:lightgray">
```

Comment:
"head" and "body" are sibling elements, both children of the "html" element. They should be indented the same amount.